app/models/models.scala

```diff
@@ -130,3 +132,16 @@ object LaunchConfiguration {
   import utils.DateUtils._
   implicit val jsonFormat = Json.format[LaunchConfiguration]
 }
+
+case class Owner(id: String, stacks: List[SSA])
+
+object Owner {
+  implicit val jsonFormat = Json.format[Owner]
+}
+
+/**
+  * @param quartzCronExpression a Quartz cron expression,
+  *                             e.g. "0 0 3 * * ? *" to run at 3am every day
+  *                             See http://www.quartz-scheduler.org/documentation/quartz-2.2.x/tutorials/crontrigger.html
+  */
+case class NotifySchedule(quartzCronExpression: String)
```

Comment:
yeah maybe, since this is only used in `NotificationScheduler` and not passed around, makes sense to remove it and only have a string. I will amend.